tests/discussions.test.js

```diff
@@ -0,0 +1,364 @@
+            response.body.should.have.property("theme");
+            response.body.should.have.property("idAccount");
+            response.body.should.have.property("dateCreation");
+            response.body.should.have.property("status");
+            idDiscussion = response.body._id;
+            done();
+        });
+    });
+
+    it("POST /Discussions Bad Request", (done) =>{
+        const discussion = {
+            title: "   ",
+            comment: " ",
+            theme: "duda",
+            idAccount: "6168cf"
+        }
+        chai.request(server).post("/discussions").send(discussion)
+        .auth(accessToken, { type: 'bearer' })
+        .end( (error, response) => {
+            response.should.have.status(400);
+            response.body.should.be.a("object");
+            response.body.should.have.property("errors");
+            done();
+        });
+    });
+
+    it("POST /Discussions Bad Request Account", (done) =>{
+        const discussion = {
+            title: "¿Qué opinan de los cuentos en zapoteco?",
+            comment: "Me gustaría saber que opinan de los cuentos que esta y donde puede encontrar más",
+            theme: "doubt",
+            idAccount: "6168cf9563929f8f000c7679"
+        }
+        chai.request(server).post("/discussions").send(discussion)
+        .auth(accessToken, { type: 'bearer' })
+        .end( (error, response) => {
+            response.should.have.status(400);
+            response.body.should.have.property("message");
+            response.body.should.have.property("message").eq("La cuenta no existe");
+            done();
+        });
+    });
+});
+
+describe("GET Tests Discussions Filters",() =>{
+    before((done) =>{
+        const login = {
+            username: "MiroStar",
+	        password: "Marst1245#"
+        };
+        chai.request(server).post("/login").send(login)
+        .end( (error, response) => {
+            accessToken = response.body.token;
+            done();
+        });
+    });
+
+    it("GET /Discussions News", (done) =>{
+        chai.request(server).get("/discussions/filters/news")
+        .auth(accessToken, { type: 'bearer' })
+        .end( (error, response) => {
+            response.should.have.status(200);
+            response.body.should.be.a("array");
+            response.body.length.should.be.eq(1);
+            done();
+        });
+    });
+
+    it("GET /Discussions Populars", (done) =>{
+        chai.request(server).get("/discussions/filters/populars")
+        .auth(accessToken, { type: 'bearer' })
+        .end( (error, response) => {
+            response.should.have.status(200);
+            response.body.should.be.a("array");
+            response.body.length.should.be.eq(1);
+            done();
+        });
+    });
+
+    after(async () => {
+        await Discussions.deleteOne({_id:idDiscussion});
+    });
+});
+
+describe("GET Tests Discussions Filters Criterion",() =>{
```

Comment:
Codacy has a fix for the issue: [Missing space after =>.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
describe("GET Tests Discussions Filters Criterion",() => {
```